src/main.cpp

```diff
@@ -56,6 +56,12 @@
 #include "type_id.h"
 #include "ui_manager.h"
 
+#if !(defined(TILES) || defined(_WIN32))
```

Comment:
Functionally speaking does this deprecate the windows notiles config?

Reply:
I was under the impression that windows notiles still uses SDL. As long as this is the case everything should still work fine. The map will render in ASCII and the ImGui windows will draw on top of them.